lib/vega_lite/wx_viewer.ex

```diff
@@ -0,0 +1,47 @@
+if Code.ensure_compiled(:wx) == {:module, :wx} do
```

Comment:
You only need to use ensure_compiled for modules within the same app. In this case, Code.ensure_loaded? is fine. you also want to check if `:wxWebView` is loaded, as it isn't available in some systems.

Reply:
If wxWebView is missing, we need to show a particular error message too.